### ISSUE TITLE
Runtime editing of values 

### ISSUE THREAD

**Issue by joshkbaker** [keijiro/KinoGlitch#17]
I found when manually editing the effects in runtime, they fail to actually show at all.

The effects become active and the value is changed yet no effect is played on the camera.

`    public IEnumerator Taking_Damage()
    {

        var volume = Character_Selection_Volume;

        if (volume.profile.TryGet<Glitch>(out var _glitch))
        {
            

            _glitch.jump = new ClampedFloatParameter(0.6f, 0, 1,true);
            _glitch.block = new ClampedFloatParameter(0.6f, 0, 1,true);

        }

        yield return null;
    }`

**Comment by keijiro** [comment]
Are you actually using this repository? Your code looks like doing something with HDRP, but this repository only contains a legacy image effect that only works on the legacy (built-in) render pipeline.

**Comment by joshkbaker** [comment]
Hi Keijiro,
I actually fixed the issue, I had to set the value in another way using the override, this ended up actually changing the effects.

That being said, I am indeed using HDRP, and this works fine? I think I have actually used a different repo and accidentally posted this issue in just the 'glitch' repo. 

Feel free to close this.

``var volume = Character_Selection_Volume;

        var Block_Speed = 0f;
        var Jitter_Speed = 0f;

        if (volume.profile.TryGet<Glitch>(out var _glitch))

{

  _glitch.block.Override(Block_Speed);
      _glitch.jitter.Override(Jitter_Speed);



}
``
